src/kibana/components/notify/_notifier.js

```diff
@@ -87,6 +87,14 @@ define(function (require) {
     return rtn;
   }
 
+  // browsers format Error.stack differently; always include message
+  function formatStack(err) {
+    if (err.stack && !~err.stack.indexOf(err.message)) {
+      return 'Error: ' + err.message + '\n' + err.stack;
+    }
+    return err.stack;
```

Comment:
If `err.stack` can be empty, I can see this leading to an empty error message. Perhaps we can use `err.message`, or `err.name`, or some combination thereof?